pkg/sensors/tracing/genericuprobe.go

```diff
@@ -142,6 +160,26 @@ func handleGenericUprobe(r *bytes.Reader) ([]observer.Event, error) {
 		unix.Args = append(unix.Args, arg)
 	}
 
+	// Cache return value on merge and run return filters below before
+	// passing up to notify hooks.
+	if uprobeEntry.loadArgs.retprobe {
```

Comment:
This is exact same code as generickprobe.go.
Couldn't find a way to make it generic enough (without making weird dances with interfaces and go generics that were at the end larger than current code) to reuse same code in both.